pages/emailSender.tsx

```diff
@@ -305,16 +334,40 @@ const EmailSender: NextPage = () => {
     )
   }
 
-  const sendEmails = () => {
+  const sendEmails = async () => {
     // format: 'FName LName <email@hackbeanpot.com>'
     const from = '' + session?.user?.name + ' <' + session?.user?.email + '>'
+    const formData = new FormData()
+    if (attachment) {
+      formData.append('file', attachment)
+    }
+
+    const uploadAttachmentsResponse = await fetch('/api/uploadAttachments', {
```

Comment:
Should all of this attachment code live inside the `if (attachment)`? I don't want to send an extra API request when we don't have any attachments

Reply:
man dean u are just too big brain 🔥